cmake/modules/Builddpdk.cmake

```diff
@@ -0,0 +1,27 @@
+if(WITH_SPDK)
```

Comment:
no need to check `WITH_SPDK`, `Builddpdk.cmake` is included only if `WITH_SPDK`.

Reply:
righ